src/components/ProductsDashboard/AddProduct/ProductForm/ProductForm.js

```diff
@@ -37,14 +38,21 @@ const renderField = ({
   meta: { touched, error },
 }) => (
   <React.Fragment>
-    <input className={`form-input${error ? ' form-error' : ''}`} {...input} type={type} />
+    <input
+      className={classNames({
+        'form-input': true,
```

Comment:
I meant
classNames('form-input', {'form-error': error})
That way form-input is added always, and form-error conditionally